lib/forklift/box_distributor.rb

```diff
@@ -0,0 +1,182 @@
+module Forklift
+  class BoxDistributor
+
+    VAGRANTFILE_API_VERSION = '2'.freeze
+
+    def initialize(boxes)
+      @ansible_groups = {}
+      @boxes = boxes
+      @boxes = @boxes.keys.sort.each_with_object({}) do |key, hash|
+        hash[key] = @boxes[key]
+        hash
+      end
+      @settings = settings
+    end
+
+    def settings
+      overrides = {}
+      settings_file = "#{File.dirname(__FILE__)}/../../settings.yaml"
+      default_settings = {
+        'memory' => 4096,
```

Comment:
Not entirely, I think my brain thought it was a strange number and rounded down to 4GB instead of 4.5GB. I can update.